python_modules/dagit/dagit/cli.py

```diff
@@ -35,12 +38,15 @@ def on_any_event(self, event):
 @repository_config_argument
 @click.option('--host', '-h', type=click.STRING, default='127.0.0.1', help="Host to run server on")
 @click.option('--port', '-p', type=click.INT, default=3000, help="Port to run server on")
-def ui(conf, host, port):
+def ui(host, port, **kwargs):
```

Comment:
Does this mean click ui won't know about those kwargs? Maybe we should have a helper that adds that to at least docs?